### DISCUSSION TITLE
Can't get the subdomain to work.

### DISCUSSION THREAD

**Discussion by kar0v** [requarks/wiki#3469] [Help / Questions]
I have a proxmox WS and on it I have a CentOS 8 with wiki.js, nginx & mariadb.

nginx is working fine on port 80, I get the standard welcome page, I also have port 3000 open for wiki.js and I can open it fine via the browser. I can also open it from another computer on the network, or outside the network with my-router-ip:3000, but I can't open it with the subdomain. I set up the subdomain with an A record in namecheap to point to my router ( I can access my site fine on port 80 ) and a URL Redirect to port 3000, but I can't access it via the subdomain name wiki.example.site. I've done the port forwarding on the router for port 80 & 3000, but the site is refusing to connect on 80 or 3000, even though I have the firewall enabled.

I have a big knowledge gap here, so I don't know how to proceed further, not even sure how to search for this. Nothing shows in /var/log/messages or in /var/log/firewalld.

**Comment by NGPixel** [comment]
This is more of a nginx configuration issue than Wiki.js...

Wiki.js doesn't care which domain it is accessed from. It will accept whatever traffic comes to its port. You shouldn't have to do URL Redirect or port forwarding if you're using nginx... nginx already handles the reverse proxy to a different port.
